lib/Target/RISCV/RISCVOperands.td

```diff
@@ -60,6 +60,24 @@ def simm13_lsb0 : Operand<OtherVT> {
   let DecoderMethod = "decodeSImmOperandAndLsl1<13>";
 }
 
+// A 8-bit unsigned immediate where the least two bits are zero.
+def uimm8_2lsb0 : Operand<i32>,
+                  ImmLeaf<i32, [{return isUInt<8>(Imm) &&
+                                        ((Imm % 4) == 0);}]> {
```

Comment:
You should able to use isShiftedUInt here

Reply:
Using isShiftedUInt which make the code much clean and clear.